gradio/components.py

```diff
@@ -4581,9 +4581,13 @@ def _postprocess_chat_messages(
         if chat_message is None:
             return None
         elif isinstance(chat_message, (tuple, list)):
-            filepath = chat_message[0]
+            image_uri = chat_message[0]
+            if utils.validate_url(image_uri):
+                filepath = image_uri
+            else:
+                filepath = self.make_temp_copy_if_needed(image_uri)
```

Comment:
```suggestion
            file_uri = chat_message[0]
            if utils.validate_url(file_uri):
                filepath = file_uri
            else:
                filepath = self.make_temp_copy_if_needed(file_uri)
```

Reply:
Just a suggestion to rename the parameter is it does not necessarily have to be an image -- can be a video or audio as well

Reply:
right, thanks!